bee-ledger/src/workers/error.rs

```diff
@@ -60,11 +60,17 @@ pub enum Error {
     #[error("invalid ledger unspent state: {0}")]
     InvalidLedgerUnspentState(u64),
     /// Consumed amount overflow.
-    #[error("consumed amount overflow: {0}")]
-    ConsumedAmountOverflow(u128),
+    #[error("consumed amount overflow")]
+    ConsumedAmountOverflow,
     /// Created amount overflow.
-    #[error("created amount overflow: {0}")]
-    CreatedAmountOverflow(u128),
+    #[error("created amount overflow.")]
```

Comment:
```suggestion
    #[error("created amount overflow")]
```
The same goes for the others as well.